.github/workflows/deploy-environment.yaml

```
@@ -0,0 +1,232 @@
+      - name: Smoke test gateway (embeddings + responses)
+        uses: ./.github/actions/smoke-test-gateway
+        with:
+          gateway_url: ${{ steps.gw.outputs.url }}
+          gateway_key: ${{ secrets.AIGATEWAY_KEY }}
+          embedding_model: ${{ env.TF_VAR_embedding_deployment }}
+          codex_model: ${{ env.TF_VAR_codex_model }}
+          aoai_endpoint: ${{ env.TF_VAR_azure_openai_endpoint }}
+          aoai_api_key: ${{ env.TF_VAR_azure_openai_api_key }}
+          max_attempts: "3"
+          retry_sleep: ${{ inputs.smoke_retry_sleep }}
+          models_wait_attempts: ${{ if(inputs.env_name == 'prod', '3', '1') }}
```

actionlint (1.7.11)

[error] undefined function "if". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)